README.md

```diff
@@ -15,16 +15,26 @@ For license information check the [LICENSE](LICENSE.md)-file.
 [![Total Downloads](https://poser.pugx.org/illuminatech/config/downloads.png)](https://packagist.org/packages/illuminatech/config)
 [![Build Status](https://travis-ci.org/illuminatech/config.svg?branch=master)](https://travis-ci.org/illuminatech/config)
 
-
-Installation
-------------
+* [Installation](#installation)
+* [Usage](#usage)
+* [Configuration Items Specification](#configuration-items-specification)
+* [Configuration Storage](#configuration-storage)
+* [Saving And Restoring Data](#saving-and-restoring-data)
+* [Caching](#caching)
+* [Validation](#validation)
+* [Creating Configuration Web Interface](#creating-configuration-web-interface)
+* [Typecast](#typecast)
+* [Encryption](#encryption)
+* [Garbage Collection](#garbage-collection)
```

Comment:
I do not think table of content make sense for such simple docs.

Anyway, this is definitely not something I am ready to support.

Reply:
Fair enough. I just found the README was lengthy and I did a lot of scrolling back and forth, and couldn't remember where certain things were found, so the list of headings felt like it would be helpful.
Nevertheless, I accept your decision.